site/pages/api/admin/delete-user.test.ts

```diff
@@ -0,0 +1,82 @@
+import { describe, it, expect, afterEach, vi } from "vitest";
+import deleteUser from "./delete-user.api";
+import { ERROR_PATH, USER_MANAGEMENT_PAGE_PATH } from "../../../constants";
+import * as cognito from "../../../data/cognito";
+import { DEFAULT_ORG_ID, getMockRequestAndResponse } from "../../../testData/mockData";
+
+describe("login", () => {
+    const writeHeadMock = vi.fn();
+
+    const deleteAdminUserSpy = vi.spyOn(cognito, "deleteUser");
+
+    const getUserDetailsSpy = vi.spyOn(cognito, "getUserDetails");
+
+    vi.mock("../../../data/cognito", () => ({
+        deleteUser: vi.fn(),
+        getUserDetails: vi.fn(),
+    }));
+
+    afterEach(() => {
+        vi.resetAllMocks();
+    });
+
+    it("should redirect to /admin/user-management if delete was a success", async () => {
```

Comment:
Can we have a test covering the case where the org ids don't match?

Reply:
done